### PR TITLE
Update New-CsOnlineSession.md

### PR DESCRIPTION
**Pull request by pupara**
Added note for TPM 2.0

### CODE REVIEW

File: skype/skype-ps/skype/New-CsOnlineSession.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ ms.reviewer:
 # New-CsOnlineSession
 
 ## SYNOPSIS
+  > [!Note]
+  > Skype for Business Online Connector is currently part of the latest Teams PowerShell module. If you're using the latest Teams PowerShell public release, you don't need to install the Skype for Business Online Connector.
+  > With Teams PowerShell Module New-CsOnlineSession has been deprecated and is no longer required to connect Skype for Business Online. It has been replaced with Connect-MicrosoftTeams.
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsOnlineSession.md:17]:
```suggestion
Note: Skype for Business Online Connector is currently part of the latest Teams PowerShell module. If you're using the latest Teams PowerShell public release, you don't need to install the Skype for Business Online Connector.
With Teams PowerShell Module New-CsOnlineSession has been deprecated and is no longer required to connect Skype for Business Online. It has been replaced with Connect-MicrosoftTeams.
```

**Review by dariomws — APPROVED** [review]
I'm ok after the note format is fixed. To avoid problems with Get-Help, we don't format notes on cmdlet articles.

Thank you!

**Review by get-itips — APPROVED** [review]

**Comment by pupara** [comment]
Can you please share the timeline when this will be live?

**Comment by get-itips** [comment]
> Can you please share the timeline when this will be live?

@Kateyanne

**Comment by Kateyanne** [comment]
@get-itips We have had an outage. I have successfully merged it. Let's check tomorrow to see if it's made it through the final publishing to live.

**Comment by pupara** [comment]
Thanks. I have few other check-ins on few other docs. I hope all of them will go live tomorrow.

Regards,
Puneet

**Pull request merged**